libtenzir/src/detail/file_path_to_parser.cpp

```diff
@@ -17,11 +17,15 @@ namespace tenzir::detail {
 constexpr inline auto fallback_parser = "json";
 
 const auto filename_to_parser_list
-  = std::array<std::pair<std::filesystem::path, std::string>, 1>{
-    {{"eve.json", "suricata"}}};
+  = std::array<std::pair<std::filesystem::path, std::string>, 1>{{
+    {"eve.json", "suricata"},
+  }};
 
 const auto extension_to_parser_list
-  = std::array<std::pair<std::string, std::string>, 1>{{{".ndjson", "json"}}};
+  = std::array<std::pair<std::string, std::string>, 2>{{
+    {".ndjson", "json"},
+    {".yml", "yml"},
+  }};
```

Comment:
What about `.yaml`? :)

Reply:
Also, the parser name `"yml"` must be changed to `"yaml"`.

Reply:
.yaml works implicitly because we first try to match the file extension. I'll change the second string to yaml, that's a good catch. I had it the other way around first, and then must've adapted this incorrectly.